test/end-to-end/offer.js

```diff
@@ -39,18 +136,39 @@ describe("Offer endpoint tests", () => {
                     expect(res.body).toHaveProperty("reason", "Invalid god token");
                 });
 
-                test("should succeed when god token is correct", async () => {
-                    const params = {};
+                test("should fail when god token is correct but owner doesn't exist", async () => {
+                    const params = { ...offer, owner: "invalidowner" };
+                    const res = await request()
+                        .post("/offers/new")
+                        .send(withGodToken(params));
+
+                    expect(res.status).toBe(HTTPStatus.UNPROCESSABLE_ENTITY);
+                    expect(res.body).toHaveProperty("error_code", ErrorTypes.VALIDATION_ERROR);
+                    expect(res.body.errors).toContainEqual({
+                        value: "invalidowner",
+                        location: "body",
+                        msg: "no-company-found-with-id-invalidowner",
```

Comment:
Not sure if we should match with the string or the `ValidatorReason.something` here

Reply:
Actually, there's no ValidatorReason.something for that, it needs to be created

Reply:
Stack it on the TODO for this PR then :stuck_out_tongue: